src/node_file.cc

```diff
@@ -479,9 +479,9 @@ void FillStatsArray(double* fields, const uv_stat_t* s) {
   fields[9] = -1;
 #endif
   // Dates.
-#define X(idx, name)                          \
-  fields[idx] = (s->st_##name.tv_sec * 1e3) + \
-                (s->st_##name.tv_nsec / 1e6); \
+#define X(idx, name)                                                 \
+  fields[idx] = (static_cast<uint32_t>(s->st_##name.tv_sec) * 1e3) + \
+                (static_cast<uint32_t>(s->st_##name.tv_nsec) / 1e6); \
```

Comment:
That was my initial thought but the linter was agree at me:
for 
`(static_cast<unsigned long>(s->st_##name.tv_sec) * 1e3)`
it spet out
`src/node_file.cc:483:  Use int16/int64/etc, rather than the C type long  [runtime/int] [4]`
got the same for:
`(unsigned long)(s->st_##name.tv_sec) * 1e3)`